ui/src/app/pages/data/data-set/export-data-set-modal.tsx

```diff
@@ -119,19 +119,28 @@ class ExportDataSetModal extends React.Component<
       cohortIds: dataSet.cohorts.map(c => c.id),
       domainValuePairs: dataSet.domainValuePairs
     };
-    await dataSetApi().exportToNotebook(
-      workspaceNamespace, workspaceFirecloudName,
-      {
-        dataSetRequest: request,
-        notebookName: this.state.notebookName,
-        newNotebook: this.state.newNotebook,
-        kernelType: this.state.kernelType
-      });
-    // Open notebook in a new tab and close the modal
-    const notebookUrl = `/workspaces/${workspaceNamespace}/${workspaceFirecloudName}/notebooks/
-      preview/${appendNotebookFileSuffix(encodeURIComponentStrict(this.state.notebookName))}`;
-    navigateByUrl(notebookUrl);
-    this.props.closeFunction();
+    try {
+      await dataSetApi().exportToNotebook(
+        workspaceNamespace, workspaceFirecloudName,
+        {
+          dataSetRequest: request,
+          notebookName: this.state.notebookName,
+          newNotebook: this.state.newNotebook,
+          kernelType: this.state.kernelType
+        });
+      // Open notebook in a new tab and close the modal
+      const notebookUrl = `/workspaces/${workspaceNamespace}/${workspaceFirecloudName}` +
+        `/notebooks/preview/${appendNotebookFileSuffix(
```

Comment:
This makes me nervous, though I think it's probably fine. Would probably unwrap this line, or just use concatenation instead of templating in this case.

Reply:
I can clean this up a bit.